backend/api/test_views.py

```diff
@@ -841,6 +841,55 @@ def test_edit_inappropriate_fields(self):
                 self.assertEqual(response.json(), expected)
 
 
+class SacFederalAwardsViewTests(TestCase):
```

Comment:
Could we add a couple of tests around auth/z? Perhaps one for an unauthenticated request, and another for a user who doesn't have access to the SAC in question

Reply:
Sure, can work on that.